babel-plugins/if-unset.js

```diff
@@ -0,0 +1,89 @@
+// Adds syntax for defaulting nullish values (null or undefined).
+// Usage:
+//   Single value
+//     IfUnset, foo = 'bar';
+//
+//   Multiple values:
+//     IfUnset, foo = 'bar', baz = 'qux';
+//
+//  Member expressions:
+//     IfUnset, foo.bar = 'baz'
+//
+//  Note that you cannot use this macro on the arguments array.
+
+module.exports = function (babel) {
```

Comment:
this is quite complex, can we make a comment about what the transform is actually doing to the code?